src/main/java/uk/ac/ebi/eva/pipeline/io/mappers/AnnotationLineMapper.java

```diff
@@ -62,29 +63,24 @@ public AnnotationLineMapper(String vepVersion, String vepCacheVersion) {
     }
 
     /**
-     * Map a line in VEP output file to {@link Annotation}
+     * Map a line in VEP output file to {@link AnnotationMongo}
      * @param line in VEP output
      * @param lineNumber
-     * @return a {@link Annotation}
+     * @return a {@link AnnotationMongo}
      *
      * Most of the code is from org.opencb.biodata.formats.annotation.io.VepFormatReader#read() with few differences:
      *  - An empty array is initialized for Hgvs (like ConsequenceTypes);
      *  - parseFrequencies is always true and the all line is always parsed;
      *  - The logic to move around the file (read line) and reference to previous line (currentVariantString) are removed;
      */
     @Override
-    public Annotation mapLine(String line, int lineNumber) {
-        ConsequenceType consequenceType = new ConsequenceType();
+    public AnnotationMongo mapLine(String line, int lineNumber) {
+        ConsequenceTypeMongo consequenceType = new ConsequenceTypeMongo(null, null,
+            null, null, null, null, null, null,
+            null, null, null, null, null, null);
```

Comment:
Why not use the Empty constructor for ConsequenceTypeMongo?

Reply:
Use of an empty constructor for ConsequenceTypeMongo results in build error - (actual and formal argument lists differ in length). This is because the default constructor isn't public.

[PR - 81](https://github.com/EBIvariation/variation-commons/pull/81) tries to address this issue. 

The error-message is as follows:  
```constructor uk.ac.ebi.eva.commons.mongodb.entities.subdocuments.ConsequenceTypeMongo.ConsequenceTypeMongo(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.Integer,java.lang.Integer,java.lang.Integer,java.lang.String,java.lang.String,uk.ac.ebi.eva.commons.mongodb.entities.subdocuments.ScoreMongo,uk.ac.ebi.eva.commons.mongodb.entities.subdocuments.ScoreMongo,java.util.Set<java.lang.Integer>,java.lang.Integer) is not applicable
(actual and formal argument lists differ in length)
```

Reply:
For reference: https://github.com/EBIvariation/variation-commons/blob/fda0d8bbf44f029ab7ef1f756f31964b3bc9e2c5/variation-commons-mongodb/src/main/java/uk/ac/ebi/eva/commons/mongodb/entities/subdocuments/ConsequenceTypeMongo.java#L99

Reply:
Hi Sundar, I was mulling over this comment and I found the following SO article useful: https://stackoverflow.com/questions/19395973/maven-adding-a-reference-to-a-parent-pom-project

The last two paragraphs in the answer are: 

"Update: You can do that in a limited way only for the dependencyManagement area with the scope import but this is only intended for using in the depdencyManagement and not for dependencies. An other solution might be to create a separate dummy project which has the dependencies too two childs as transitive deps. But this is not really beauty.

So the solution is simply to add the two or more dependencies directly which you are using in you project and that's it.

Parent POM isn't for that. To create a meta-project, create a regular Maven project with a name like common-libraries and reference all of your other projects as dependencies. Parent POM is to build several libraries in one step, and automatically use the correct build order."

What this suggests is to add the the dependencies directly. I just wanted to know how this can be resolved by tonight or tomorrow.